remote-query/remote-query-server/src/main/java/org/infinispan/query/remote/impl/ObjectRemoteQueryEngine.java

```diff
@@ -22,13 +20,8 @@
 
    private final EmbeddedQueryFactory queryFactory = new EmbeddedQueryFactory(this);
 
-   ObjectRemoteQueryEngine(AdvancedCache<?, ?> cache, boolean isIndexed, Class<? extends Matcher> matcherImplClass,
-                           LuceneQueryMaker.FieldBridgeAndAnalyzerProvider<Descriptor> fieldBridgeAndAnalyzerProvider) {
-      super(cache, isIndexed, matcherImplClass, fieldBridgeAndAnalyzerProvider);
-   }
-
    ObjectRemoteQueryEngine(AdvancedCache<?, ?> cache, boolean isIndexed, Class<? extends Matcher> matcherImplClass) {
-      super(cache.getAdvancedCache().withEncoding(IdentityEncoder.class), isIndexed, matcherImplClass, null);
+      super(cache, isIndexed, matcherImplClass);
```

Comment:
Why was `withEncoding(IdentityEncoder.class)` removed? Note sure what this implies.

Reply:
Good catch... probably it implies something bad, but I've not restored a test for this.
Anyway, I've restored as it was.